src/mcp/client/streamable_http.py

```diff
@@ -40,6 +40,7 @@
 GetSessionIdCallback = Callable[[], str | None]
 
 MCP_SESSION_ID = "mcp-session-id"
+MCP_PROTOCOL_VERSION = "MCP-Protocol-Version"
```

Comment:
capitalization on these headers is inconsistent across the codebase, should we align on `kebab-case-everywhere` for headers?

Reply:
```suggestion
MCP_PROTOCOL_VERSION = "mcp-protocol-version"
```